### DISCUSSION TITLE
Hash Navigation dont work if enable free mode

### DISCUSSION THREAD

**Discussion by Asim-Tahir** [nolimits4web/swiper#4478] [Q&A] [answered]
- Swiper Version: 6.5.8
- Platform/Target and Browser Versions: Chrome: 90.0.4430.85

## What You Did

Enable free mode and hash navigation in same time. Free mode work very well. But hash navigation not working. If enable freeMode and freeModeSticky, hash navigation work very well. But in my case i cant use freeModeSticky. Cuz slides vertically overflow the screen. freeModeSticky makes hard to read content.

https://codesandbox.io/s/swiper-hash-navigation-free-mode-repro-jx78m

## Expected Behavior

Hash Navigation should be work if free mode enabled. Without freeModeSticky property.

## Actual Behavior

If enable free mode, hash navigation dont work.

**Comment by vltansky** [comment, marked as answer]
closed by https://github.com/nolimits4web/swiper/commit/c90a7bd73dd84b4422a5699ffb996f24ec868d01

**Reply by Asim-Tahir** [reply]
When the closest release date 👀
